src/Validation/FrValidation.php

```diff
@@ -46,7 +46,7 @@ public static function postal($check)
         if ((bool)preg_match($pattern, $check)) {
             $value = intval($check);
 
-            return $value >= 1000 && $value <= 99138;
+            return $value >= 1001 && $value <= 99138;
```

Comment:
@2ndGAB Alain, this value should not change, if you can't update i can do it post merge